### PR TITLE
Switch Windows build to Qt-5.9

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: mscore/CMakeLists.txt [hunk 1]
```diff
@@ -475,7 +475,6 @@ if (MINGW)
       ${CROSS}/lib/portaudio.dll
       ${CROSS}/opt/bin/libeay32.dll
       ${CROSS}/opt/bin/ssleay32.dll
-      ${CROSSQT}/bin/Qt5CLucene.dll
```
Comment (Jojo-Schmitz) [inline mscore/CMakeLists.txt]: A comment furter up implies that build/Linux+BSD/portable/copy-libs needs to get adjusted too?

**Comment by lasconic** [comment]
Qt 5.9.1 is due this month. I just switched the mac build on travis to 5.9.0 but the difference between mac/linux and windows build is that they use whatever Qt they find with qmake... and so we can change the version on travis without impacting developers. Since it's not needed, I will wait a bit to merge this one.

**Comment by Jojo-Schmitz** [comment]
Fine by me ;-)

**Comment by Jojo-Schmitz** [comment]
I guess #3215  makes this PR here obsolete?

**Comment by Jojo-Schmitz** [comment]
Qt 5.0.2 is due next week

**Comment by Jojo-Schmitz** [comment]
Qt-5.9.2 is out now

File: build/mingw32.mingw.cmake [hunk 2]
```diff
@@ -5,7 +5,7 @@
 set (CMAKE_SYSTEM_NAME "Windows")
 
 set (CROSS C:/Qt/Tools/mingw530_32)
-set (CROSSQT C:/Qt/5.8/mingw53_32)
+set (CROSSQT C:/Qt/5.9/mingw53_32)
```
Comment (Jojo-Schmitz) [inline build/mingw32.mingw.cmake]: This got to be `set (CROSSQT C:/Qt/5.9.2/mingw53_32) ` for Qt-5.9.2 now

**Comment by Jojo-Schmitz** [comment]
Qt-5.9.3 will come out very shortly

**Comment by Jojo-Schmitz** [comment]
Qt-5.9.3 just got released

**Comment by lasconic** [comment]
this will be taken care differently. See PR #3213

**Pull request closed without merging**